doc-content/jbpm-docs/src/main/asciidoc/Integration/ejb-interfaces-local-ref.adoc

```diff
@@ -0,0 +1,31 @@
+[id='ejb-interfaces-local-ref_{context}']
+= Local EJB interfaces
```

Comment:
I would say this should be a procedure module.  In any case, try to think of modules as standalone. Does it contain enough information to be understood outside the context of the assembly. In this case, I don't think it does. I would make this a procedure module and add more context.